README.md

```diff
@@ -1,3 +1,49 @@
-### Ronin Puppet
-#### The masterless puppet collection
+# Ronin Puppet: the masterless puppet collection
 [![Build Status](https://travis-ci.com/mozilla-platform-ops/ronin_puppet.svg?branch=master)](https://travis-ci.com/mozilla-platform-ops/ronin_puppet)
+
+## structure
+
+```
+modules/roles_profiles/manifests/roles
+modules/roles_profiles/manifests/profiles
+```
+
+Roles specify everything a machine type needs to fufill a role.
+
+Profiles provide an OS-independent interface to functionality provided by modules.
+
+## testing
```

Comment:
thankyou for the documentation. I know nothing about kitchen, and this shows me how to start using it

Reply:
For sure. :D